shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1197,4 +1197,87 @@ - (NSObject*)valuePublishedByPlugin:(NSString*)pluginKey {
   return [_engine.get() valuePublishedByPlugin:pluginKey];
 }
 
+- (void)viewWillTransitionToSize:(CGSize)size
+       withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator {
+  [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
+
+  // On transition behavior on iOS and how Flutter tries to reconcile differences.
+  // -----------------------------------------------------------------------------
+  //
+  // The point of this method is to hide rendering artifacts as the old backing store for the
+  // renderer is being torn down and recreated at the new size. Since Flutter won't generating
+  // frames in transitionary state states, snpashotted contents have to interpolated instead.
+  //
+  // iOS seems the layout the view in the final state after the transition, snapshot it, and animate
```

Comment:
`iOS sees`?

Reply:
`snapshots` `animates`?

Reply:
"iOS seems to layout the". Fixed. Also fixed the rest.